pkg/core/blockchain.go

```diff
@@ -552,6 +559,34 @@ func (bc *Blockchain) storeBlock(block *block.Block) error {
 					}
 					break
 				}
+
+				tx, _, err := cache.GetTransaction(input.PrevHash)
+				if err != nil || int(input.PrevIndex) > len(tx.Outputs) {
+					return err
```

Comment:
You may return `nil` here and I'm not sure it's the desired behavior.

Reply:
Fixed.